### ISSUE TITLE
Batch Normalization is commented out

### ISSUE THREAD

**Issue by SiyingXu** [midas-tum/merlin#4]
The batch normalization is commented out in 'complex norm':
master/tensorflow/merlintf/keras/layers/complex_norm.py

If I can use it by directly remove the comments?
